### DISCUSSION TITLE
roadmap and wishlist

### DISCUSSION THREAD

**Discussion by axetroy** [vlang/v#9245] [General]
I suggest that the question should be a roadmap and wishlist

The roadmap is used to let everyone know the progress of the project and the future direction

The wishlist is used to list the features that vlang hopes to add, but there is currently no.

I think this may help those who want to contribute to vlang to have a point of participation :)

eg.

### wishlist

- compile
  - [ ] cross compile
- checker
  - [ ] some feature
- standard library
  - [ ] os/glob

**Comment by fossdd** [comment]
Umm is not here already a roadmap: https://github.com/vlang/v/blob/master/ROADMAP.md?
And here the wishlist: https://github.com/vlang/v/discussions/7610
